### PR TITLE
qrenderdoc Android launch, capture & open now works.

### PR DESCRIPTION
**Pull request by michaelrgb**

### CODE REVIEW

File: renderdoc/driver/gl/gl_debug.cpp [hunk 1]
```diff
@@ -3762,7 +3763,9 @@ void GLReplay::InitPostVSBuffers(uint32_t eventID)
   GLuint vsoutBuffer = 0;
   gl.glGenBuffers(1, &vsoutBuffer);
   gl.glBindBuffer(eGL_ARRAY_BUFFER, vsoutBuffer);
-  gl.glNamedBufferDataEXT(vsoutBuffer, stride * primsWritten, data, eGL_STATIC_DRAW);
+  GLsizeiptr size = stride * primsWritten;
+  gl.glBufferData(eGL_ARRAY_BUFFER, size, data, eGL_STATIC_DRAW);
+  gl.m_Buffers[gl.GetResourceManager()->GetID(BufferRes(gl.GetCtx(), vsoutBuffer))].size = size;
```
Comment (michaelrgb) [inline renderdoc/driver/gl/gl_debug.cpp]: This needs moving somewhere.

**Comment by baldurk** [comment]
Can you split the Map change out into a separate PR? It makes it easier to test/manage if they're in logical groups together.

**Comment by michaelrgb** [comment]
Done.

**Comment by baldurk** [comment]
Merged manually.

**Pull request closed without merging**